src/vs/workbench/contrib/files/browser/files.contribution.ts

```diff
@@ -323,6 +323,11 @@ configurationRegistry.registerConfiguration({
 			'default': 'askUser',
 			'scope': ConfigurationScope.LANGUAGE_OVERRIDABLE
 		},
+		'files.dialog.homePath': {
```

Comment:
I wonder if the `default` of this setting should be filled in based on the `IPathService.userHome`

Reply:
@bpasero I'm unclear what you're suggesting. Currently if the user leaves this machine-level string setting unset or sets it to "" in JSON they will continue getting the existing behaviour.

Reply:
I think its fine. My idea was to show to the user the default path that we pick based on the environment.